samples/boards/olimex_stm32_e407/ccm/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _olimex_stm32_e407_ccm:
+
+Olimex STM32-E407 CCM example
+#############################
+
+Overview
+********
+
+Show usage of the Core Coupled Memory (CCM) that is available
+on several STM32 devices. The very important difference with
+normal RAM is that CCM can not be used for DMA.
+
+By prefixing a variable with __ccm_data_section, __ccm_bss_section,
+or __ccm_noinit_section those variables are placed in the CCM.
+
+The __ccm_data_section prefix should be used for variables that
+are initialized. Like the normal data section the initial
+values take up space in the FLASH image.
+
+The __ccm_bss_section prefix should be used for variables that
+should be initialized to 0. Like the normal bss section they
+do not take up FLASH space.
+
+The __ccm_noinit_section prefix should be used for variables
+that don't need to have a defined initial value (for example
+buffers that will receive data). Compared to bss or data the
+kernel does not need to initialize the noinit section making
+the startup slightly faster.
+
+
+Building
+********
+
+.. code-block:: console
+
+   $ cd samples/boards/olimex_stm32_e407/ccm
+   $ make
```

Comment:
Could you also add a "Running" section.  I see the sample writes some output that might be worth showing and explaining here.

Reply:
Good idea, that should be no problem.